.changeset/eighty-donkeys-attack.md

```diff
@@ -0,0 +1,6 @@
+---
+'eslint-plugin-next-on-pages': patch
+'@cloudflare/next-on-pages': patch
+---
+
+chore: Bump dependencies
```

Comment:
does dependency bumping require changesets? 😮

Reply:
Yes. Otherwise it won't trigger a release. And it's something that a user might care about because this could conceivably cause a bug in the next release, so this would help them narrow down where the problem was introduced.

Reply:
ah ok, thanks 🙂, I'll keep that in mind, I think I haven't included them in the changesets till now 😓 👍